src/aerovaldb/jsondb/jsonfiledb.py

```diff
@@ -467,6 +468,7 @@ async def get_heatmap(
         )
 
     @async_and_sync
+    @alru_cache
```

Comment:
Since you have no control over 'self', this cache can grow forever. Please add a `(maxsize=xxx)` with xxx something >> maximum number of file_path (100 or 1000?)